docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst

```diff
@@ -0,0 +1,100 @@
+- **Floaters** - Other duties as assigned ;) Assisting with odds and ends, requests from other volunteers and organizers, as well as answering questions; overall, being generally helpful
+
+If you're not sure how to do any of these tasks, don't worry, the organizers will be available for questions and any problems you may encounter. We also have a dedicated volunteer coordinator and some volunteers from previous years will join us again, so you will have support.
+
+To register as a volunteer for the conference, please read and fill out the `Call for Volunteers form <https://forms.gle/LVujT6TBdt3DzmeM8>`_. You will receive additional information by mid-July regarding shifts, detailed duties, and other updates to the volunteer duties and schedules.
+
+Please email our volunteer coordinator Sasha Romijn at sasha@writethedocs.org if you have any questions about volunteering for the conference.
+
+Tickets are going fast!
+-----------------------
+
+If you don't have your ticket yet, now's a great time to snap one up. Last year we reached capacity nearly one month before the conference, and this year we expect to reach it even sooner, so if you're planning to come, don't leave it to the last minute.
+
+`Buy your tickets here! <https://www.writethedocs.org/conf/prague/2019/tickets/>`_
+
+Diversity tickets reminder
+--------------------------
+
+If you are still considering whether to apply for our new diversity ticket program, head on over to the `DiversityTickets.org Event <https://diversitytickets.org/en/events/425/>`_ page and check it out. This program is run in collaboration with the wonderful folks at **DiversityTickets.org**, a non-profit organization that shares qualifying events with various diversity organizations and helps people from underrepresented groups find events to attend.
+
+Diversity scholarship tickets are **free**, and we hope to welcome more people from all walks of life to join our community. The deadline for applications is **7th July**.
+
+Please share this campaign with your social networks and encourage folks to apply if they are interested to join us!
+
+Conference shirts
+-----------------
+
+Following the success of our pilot last year, once again we'll be selling t-shirts separately from your conference ticket. This is due partially to the logistics of sizing, ordering, and managing stock, but also to help reduce waste from the inevitable leftovers that we find ourselves with every year.
+
+Head over to our Write the Docs Prague 2019 `Pop-Up Shop <https://teespring.com/wtd-prague-2019>`__ and order this year's shirt. We have several cuts and fabrics available, and we believe that you'll be able to find the size and style that fits you. The campaign will run until **30th September** in case anyone wants to order their shirts after the conference, but if you want to sport your new threads at the conference, we recommend to order at least 2-3 weeks in advance.
```

Comment:
Is "sport your new threads" a sufficiently well-known expression?

Reply:
I don't think there's a problem with it, do you have strong feelings about it?

Reply:
No strong feelings. If you think it's fine, let's leave it in.